script.js

```diff
@@ -1,12 +1,77 @@
 //You can edit ALL of the code here
+
 function setup() {
   const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
-
+//Level 100
 function makePageForEpisodes(episodeList) {
   const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  episodeList.forEach((element) => {
+    //creating DOM elements
+    let divElement = document.createElement("div");
+    let headerElement = document.createElement("h3");
+    let imgElement = document.createElement("img");
+    let pElement = document.createElement("p");
+    let aElement = document.createElement("a");
+
+    //giving them a class
+    divElement.classList.add("single-container");
+
+    //assigning values to DOM elements
+    headerElement.innerHTML = `${element.name} - S${String(
+      element.season
+    ).padStart(2, "0")}E${String(element.number).padStart(2, "0")}`;
```

Comment:
Nitpick: It's a little difficult to read these lines. I'd suggest splitting this up to make it more readable.